app/controllers/TagManagementApi.scala

```diff
@@ -227,12 +233,14 @@ object TagManagementApi extends Controller with PanDomainAuthActions {
     }
   }
 
-  def deleteTag(id: Long) = (APIAuthAction andThen DeleteTagPermissionsCheck).async { req =>
-    implicit val username = Option(req.user.email)
-    (new DeleteTagCommand(id)).process.map{ result =>
-      result.map{t => NoContent } getOrElse NotFound
-    } recover {
-      commandErrorAsResult
+  def deleteTag(id: Long) =  CORSable(conf.corsablePostDomains: _*) {
+    (APIHMACAuthAction andThen DeleteTagPermissionsCheck).async { req =>
+      implicit val username = Option(req.user.email)
+      (new DeleteTagCommand(id)).process.map { result =>
```

Comment:
Do you need the `new` keyword here for `DeleteTagCommand`? Is it a case class or a class?

Reply:
it is indeed a case class, so doesn't need `new`